ID.me WebVerify SDK/IDmeWebVerify.m

```diff
@@ -130,21 +133,59 @@ -(void)registerConnectionInViewController:(UIViewController *)viewController
     [self launchWebNavigationControllerWithDelegate:connectionDelegate completion:^{
 
         // Register Connection via UIWebView flow
-        [self loadWebViewWithConnectionRegisterRequestPage];
+        [weakself getAccessTokenWithScope:requestScope
+                          forceRefreshing:NO
+                                   result:^(NSString * _Nullable accessToken, NSError * _Nullable error) {
```

Comment:
I think even without token we should go on. The user will have to log in but nothing more.